### DISCUSSION TITLE
建议

### DISCUSSION THREAD

**Discussion by Robert-Stackflow** [syhyz1990/baiduyun#65] [Ideas]
新版的百度网盘网页没有了下载助手按钮，以下是新版网页https://pan.baidu.com/disk/main?from=oldversion#/index?category=all

**Comment by syhyz1990** [comment]
@Robert-yutuan 感谢建议，新版界面已经支持了哦！更新到最新版即可 https://www.baiduyun.wiki/install.html?from=update
